src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -9746,7 +9782,8 @@ private ExpressionSyntax ParseCastOrParenExpressionOrLambdaOrTuple(Precedence pr
                     //  ( <expr>,    must be a tuple
                     if (this.CurrentToken.Kind == SyntaxKind.CommaToken)
                     {
-                        var firstArg = _syntaxFactory.Argument(nameColon: null, refOrOutKeyword: default(SyntaxToken), expression: expression);
+                        var firstArg = _syntaxFactory.Argument(nameColon: null, refOrOutKeyword: default(SyntaxToken), expression: expression, 
```

Comment:
Same response.

---
In reply to: [63745716](https://github.com/dotnet/roslyn/pull/11393#discussion_r63745716) [](ancestors = 63745716)